docs/reference/run.md

```diff
@@ -1260,6 +1260,10 @@ one or more `VOLUME`'s associated with an image, but only the operator
 can give access from one container to another (or from a container to a
 volume mounted on the host).
 
+> **Note:** 
```

Comment:
This shouldn't be a note. It isn't exception it is how the command works.  Also, would benefit from an example.

---> https://gist.github.com/moxiegirl/d64149fcf7784c03807c

The `host-dir` can be relative or absolute. This means both `/foo` or `foo` can be a `host-dir` value.   The `container-dir` must be an absolute path such as `/src/docs`.